File: 06_map_values.py
#!/usr/bin/env python3

# Python 3.9.5

# 06_map_values.py

# Dependency
import random

data = [random.randint(1, 31) for i in range(20)]
smaller_ten = [value < 10 for value in data]
print(smaller_ten)

File: 06_map_values.js
// 06_map_values.js

const data = [39, 10, 26, 7, 13, 19, 25, 8, 14, 29, 31, 3];
const smaller_ten = data.map(function(value) {
  return value < 10; 
})

document.write(smaller_ten);
